translations/catroweb.en.yml

```diff
@@ -66,6 +67,11 @@ notificationsClearError: "Unable to mark notifications as read!"
 notificationsDeleteError: "Unable to delete notifications!"
 pleaseWait: "Please wait..."
 projects: "Projects"
+oauthPopUpTitle: "Oauth Sign In Information"
+oauthInfo: 'You have just used Oauth system to sign in. We created a random username for you. If you want
```

Comment:
Do user understand oauth? or should we think of a term even kids understand?
any ideas?

Reply:
the oauthInfo.. where is this information shown? I have not seen it once while testing

Reply:
Yep, external should be fine I guess